src/main/webapp/newReview.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>newReview</title>
+    <link rel="stylesheet" href="newReview.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="index-script.js"></script>
+    <script src="api.js"></script>
+  </head>
+  <body onload="loadPlaceID()">
+    <div class="container">
+      <form action="/review" method="POST">
+        <div class="review-row">
+          <input type="hidden" id="place_id" name="place_id" display="none">
+        </div>
+        <div class="review-row">
+          <div class="col">
+            <label for="fname">First Name</label>
+            <input type="text" id="fname" name="firstname" placeholder="Your name..">
```

Comment:
No need to address in this PR. Just some thoughts:

Can users set any custom names? or is it unique to an email address?
Should this be a username/nickname instead? First/last names seem rigid.